### ISSUE TITLE
CONTROL + Backspace in Nushell calls for CONTROL + H

### ISSUE THREAD

**Issue by maxim-uvarov** [wez/wezterm#3489]
I'm really sorry to bother you with such a minor issue, but I wanted to bring it to your attention in case it can be easily fixed. If it cannot, please feel free to ignore or close it. I can live with this issue.

### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

wezterm 20230408-112425-69ae8472

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

No, and I'll explain why below

### Describe the bug

when I press `ctrl + backspace` on my mac - nushell gives me slow menu of previous commands. 
`keybindings listen` gives me that I type `ctrl + h`. 
I don't want to have any menu on ctrl + backspace, how can I disable it?

<img width="674" alt="image" src="https://user-images.githubusercontent.com/4896754/230856557-68c025fb-5cc3-4e63-bc61-4044d1a8c138.png">

![image](https://user-images.githubusercontent.com/4896754/230855807-645af361-1c65-4788-a427-565d4e27d403.png)

### Configuration

```
local wezterm = require 'wezterm'

-- Allow working with both the current release and the nightly
local config = {}
if wezterm.config_builder then
  config = wezterm.config_builder()
end


config.default_prog = { '/Users/user/.cargo/bin/nu', '-l'}
-- config.default_prog = { '/Users/user/.cargo/bin/nu077dfr', '-l', '-t', '30' }
-- config.debug_key_events = true


config.font = wezterm.font { family = 'JetBrains Mono' }
config.font_size = 16.0
config.initial_cols = 120
config.initial_rows = 120
config.skip_close_confirmation_for_processes_named = {
  'bash',
  'sh',
  'zsh',
  'fish',
  'tmux',
  'nu',
  'nu074',
  'nu076',
  'nu076df',
  'nu077',
  'nu077dfr'
}
config.scrollback_lines = 10000
config.keys = {
  -- Turn off the default CMD-m Hide action, allowing CMD-m to
  -- be potentially recognized and handled by the tab
  {
    key = 'Enter',
    mods = 'ALT',
    action = wezterm.action.DisableDefaultAssignment,
  },
  {
    key = 'Backspace',
    mods = 'CTRL',
    action = wezterm.action.DisableDefaultAssignment,
  }
}
config.mouse_wheel_scrolls_tabs = false
-- conifg.quick_select_patterns = {
--   "[0-9A-F]{64}",
-- }
config.debug_key_events = true     
config.launch_menu = {
  {
    args = { 'top' },
  },
  {
    -- Optional label to show in the launcher. If omitted, a label
    -- is derived from the `args`
    label = 'nu multithreads',
    -- The argument array to spawn.  If omitted the default program
    -- will be used as described in the documentation above
    args = { 'nu', '-l', '-t', '40' },

    -- You can specify an alternative current working directory;
    -- if you don't specify one then a default based on the OSC 7
    -- escape sequence will be used (see the Shell Integration
    -- docs), falling back to the home directory.
    -- cwd = "/some/path"

    -- You can override environment variables just for this command
    -- by setting this here.  It has the same semantics as the main
    -- set_environment_variables configuration option described above
    -- set_environment_variables = { FOO = "bar" },
  },
}

return config
```

### Logs

```
07:59:05.473  INFO   wezterm_gui::termwindow::keyevent > key_event RawKeyEvent { key: Char('\u{8}'), modifiers: CTRL, phys_code: Some(Backspace), raw_code: 51, repeat_count: 1, key_is_down: true, handled: Handled(false) }
07:59:05.474  INFO   wezterm_gui::termwindow::keyevent > key_event KeyEvent { key: Char('\u{8}'), modifiers: CTRL, repeat_count: 1, key_is_down: true, raw: Some(RawKeyEvent { key: Char('\u{8}'), modifiers: CTRL, phys_code: Some(Backspace), raw_code: 51, repeat_count: 1, key_is_down: true, handled: Handled(false) }) }
07:59:05.474  INFO   wezterm_gui::termwindow::keyevent > send to pane DOWN key=Backspace mods=CTRL
07:59:05.474  INFO   wezterm_term::terminalstate::keyboard > key_down: sending "\u{8}", Backspace CTRL
07:59:05.587  INFO   wezterm_gui::termwindow::keyevent     > key_event RawKeyEvent { key: Char('\u{8}'), modifiers: CTRL, phys_code: Some(Backspace), raw_code: 51, repeat_count: 1, key_is_down: false, handled: Handled(false) }
07:59:05.587  INFO   wezterm_gui::termwindow::keyevent     > key_event KeyEvent { key: Char('\u{8}'), modifiers: CTRL, repeat_count: 1, key_is_down: false, raw: Some(RawKeyEvent { key: Char('\u{8}'), modifiers: CTRL, phys_code: Some(Backspace), raw_code: 51, repeat_count: 1, key_is_down: false, handled: Handled(false) }) }
07:59:05.587  INFO   wezterm_gui::termwindow::keyevent     > send to pane UP key=Backspace mods=CTRL
```

### Anything else?

I tried to use 

```
config.keys = {
  {
    key = 'Backspace',
    mods = 'CTRL',
    action = wezterm.action.DisableDefaultAssignment,
  }
}
``` 
with no luck.

Here is my configs of nushell:

```
  keybindings: [
    {
      name: completion_menu
      modifier: none
      keycode: tab
      mode: [emacs vi_normal vi_insert]
      event: {
        until: [
          { send: menu name: completion_menu }
          { send: menunext }
        ]
      }
    }
    {
      name: completion_previous
      modifier: shift
      keycode: backtab
      mode: [emacs, vi_normal, vi_insert] # Note: You can add the same keybinding to all modes by using a list
      event: { send: menuprevious }
    }
    # {
    #   name: history_menu
    #   modifier: control
    #   keycode: char_r
    #   mode: emacs
    #   event: { send: menu name: history_menu }
    # }
    {
      name: next_page
      modifier: control
      keycode: char_x
      mode: emacs
      event: { send: menupagenext }
    }
    {
      name: undo_or_previous_page
      modifier: control
      keycode: char_z
      mode: emacs
      event: {
        until: [
          { send: menupageprevious }
          { edit: undo }
        ]
       }
    }
    {
      name: yank
      modifier: control
      keycode: char_y
      mode: emacs
      event: {
        until: [
          {edit: pastecutbufferafter}
        ]
      }
    }
    {
      name: unix-line-discard
      modifier: control
      keycode: char_u
      mode: [emacs, vi_normal, vi_insert]
      event: {
        until: [
          {edit: cutfromlinestart}
        ]
      }
    }
    {
      name: kill-line
      modifier: control
      keycode: char_k
      mode: [emacs, vi_normal, vi_insert]
      event: {
        until: [
          {edit: cuttolineend}
        ]
      }
    }
    # Keybindings used to trigger the user defined menus
    {
      name: commands_menu
      modifier: control
      keycode: char_t
      mode: [emacs, vi_normal, vi_insert]
      event: { send: menu name: commands_menu }
    }
    {
      name: vars_menu
      modifier: alt
      keycode: char_o
      mode: [emacs, vi_normal, vi_insert]
      event: { send: menu name: vars_menu }
    }
    # {
    #   name: vars_menu
    #   modifier: control
    #   keycode: char_o
    #   mode: [emacs, vi_normal, vi_insert]
    #   event: { send: menu name: vars_menu }
    # }
    {
      name: commands_with_description
      modifier: control
      keycode: char_s
      mode: [emacs, vi_normal, vi_insert]
      event: { send: menu name: commands_with_description }
    }
    {
      name: "all history"
      modifier: control
      keycode: char_r
      mode: emacs
      event: { send: menu name: all_history_menu }
    }
    {
      name: "pwd history"
      modifier: control
      keycode: char_h
      mode: emacs
      event: { send: menu name: pwd_history_menu }
    }

    {
      name: "current_session_menu"
      modifier: alt
      keycode: char_r
      mode: emacs
      event: { send: menu name: current_session_menu }
    }

    {
      name: "fzf_menu_nu_ui"
      modifier: alt
      keycode: char_f
      mode: emacs
      event: { send: menu name: fzf_menu_nu_ui }
    }

    {
      name: fuzzy_history
      modifier: control
      keycode: char_f
      mode: [emacs , vi_normal, vi_insert]
      event: {
        send: executehostcommand
        cmd: "commandline -i (
          history | each { |it| $it.command | str replace '\n' ' '} | uniq 
          | reverse | str join (char nl) 
          | fzf --tiebreak=chunk --layout=reverse  --multi --preview='echo {..}' --preview-window='bottom:3:wrap' --height=70% 
          | decode utf-8 | str trim)"
      }
    }

    {
      name: temp_var
      modifier: alt
      keycode: char_v
      mode: [emacs , vi_normal, vi_insert]
      event: {
        send: executehostcommand
        cmd: "let-env temp_var = ($env | get -i temp_var | default 0 | $in + 1); 
        commandline ('let t' + ($env.temp_var | into string) + ' = (' + (commandline) + ')')"
      }
    }
  ]
```

**Comment by wez** [comment]
Maybe you want: https://wezfurlong.org/wezterm/config/lua/config/swap_backspace_and_delete.html

**Comment by maxim-uvarov** [comment]
> Maybe you want: https://wezfurlong.org/wezterm/config/lua/config/swap_backspace_and_delete.html

Thank you for your response! I tried this option, and the menu no longer appears when using the shortcut "Ctrl + Backspace". However, I noticed that the behavior of the backspace button is inconvinient now.

I forgot to mention that 'Ctrl + Backspace' shortcut in Alacritty and iTerm also do not trigger the menu, so it seems to be related to Wezterm. Nonetheless, I can manage without it. Thank you!
